SQL/0000-00-00-schema.sql

```diff
@@ -852,6 +873,50 @@ INSERT INTO bids_mri_scan_type_rel
     NULL
   );
 
+
+CREATE TABLE `bids_export_file_level_category` (
+  `BIDSExportFileLevelCategoryID`   int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `BIDSExportFileLevelCategoryName` varchar(12) NOT NULL,
+  PRIMARY KEY (`BIDSExportFileLevelCategoryID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO bids_export_file_level_category (BIDSExportFileLevelCategoryName) VALUES
+  ('study'),
+  ('image'),
+  ('session');
+
+CREATE TABLE `bids_export_non_imaging_file_category` (
+  `BIDSNonImagingFileCategoryID`   int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `BIDSNonImagingFileCategoryName` varchar(40) NOT NULL,
+  PRIMARY KEY (`BIDSNonImagingFileCategoryID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO bids_export_non_imaging_file_category (BIDSNonImagingFileCategoryName) VALUES
+  ('dataset_description'),
+  ('README'),
+  ('bids-validator-config'),
+  ('participants_list_file'),
+  ('session_list_of_scans');
+
+CREATE TABLE `bids_export_files` (
+  `BIDSExportedFileID`           int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `BIDSExportFileLevelID`        int(10) unsigned NOT NULL,
+  `FileID`                       int(10) unsigned DEFAULT NULL,
+  `SessionID`                    int(10) unsigned DEFAULT NULL,
+  `BIDSNonImagingFileCategoryID` int(10) unsigned DEFAULT NULL,
+  `BIDSCategoryID`               int(3)  unsigned DEFAULT NULL,
+  `FileType`                     varchar(12) NOT NULL,
+  `FilePath`                     varchar(255) NOT NULL,
+  PRIMARY KEY (`BIDSExportedFileID`),
+  CONSTRAINT `FK_bef_BIDSExportFileLevelID`        FOREIGN KEY (`BIDSExportFileLevelID`)        REFERENCES `bids_export_file_level_category` (`BIDSExportFileLevelCategoryID`),
```

Comment:
@cmadjar What is the reason the name of the attribute is `BIDSExportFileLevelID` in this table and refer to `BIDSExportFileLevelCategoryID`?

Reply:
@cmadjar  ping!

Reply:
@PapillonMcGill Thank you! Corrected. Not sure why I did that in the first place...